ccd-definition/CaseField.json

```diff
@@ -187,6 +187,23 @@
     "FieldType": "Applicant",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "applicants",
+    "Label": "Applicant",
```

Comment:
Shall that be singular form if more that one is allowed?

Reply:
👍